package_control/package_installer.py

```diff
@@ -50,15 +50,23 @@ def make_package_list(self, ignore_actions=[], override_action=None, ignore_pack
         :param ignore_packages:
             A list of packages names that should not be returned in the list
 
+        :param get_dependencies:
+            Whether to return only a list of dependencies. By default only normal packages
+            are returned.
+
         :return:
             A list of lists, each containing three strings:
               0 - package name
               1 - package description
               2 - action; [extra info;] package url
         """
 
-        packages = self.manager.list_available_packages()
-        installed_packages = self.manager.list_packages()
+        if get_dependencies:
+            packages = self.manager.list_available_dependencies()
+            installed_packages = self.manager.list_dependencies()
```

Comment:
These functions have been renamed on the four-point-oh branch.

https://github.com/wbond/package_control/blob/79c6275f1fdc28ad3aa9d1939a6d94dedc9c6c20/package_control/package_manager.py#L175

Reply:
I did some renaming to my changes but no, those are still not renamed on four-point-oh branch.

Reply:
You're right, I got the wrong function there.